client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx

```diff
@@ -19,19 +18,19 @@ export const tabs: TabObj[] = [
         id: 1,
         name: 'בידוד ופרטים קליניים',
         isDisabled: false,
-        displayComponent: <></>,
+        displayComponent: <></>
     },
     {
         id: 2,
         name: 'חשיפה אפשרית וחו"ל',
         isDisabled: false,
-        displayComponent: <></>,
+        displayComponent: <></>
     },
     {
         id: 3,
-        name: 'מקומות ומגעים', 
-        displayComponent: <InteractionsTab/>,
+        name: 'מקומות ומגעים',
         isDisabled: false,
+        displayComponent: <ContactsAndExposuresTab />
```

Comment:
This is not the proper component, you need to use Maya’s InteractionsTab component and nest your component inside it